.gitignore

```diff
@@ -14,7 +14,7 @@ dist/
 downloads/
 eggs/
 .eggs/
-lib/
+# lib/
```

Comment:
You should remove this rather than comment it out

Reply:
Done